custodian/vasp/jobs.py

```diff
@@ -162,20 +167,24 @@ def setup(self):
             except:
                 pass
 
-        # Auto continue if a read-only STOPCAR is present
-        if self.auto_continue and \
-           os.path.exists("STOPCAR") and \
-           not os.access("STOPCAR", os.W_OK):
-            # Remove STOPCAR
-            os.chmod("STOPCAR", 0o644)
-            os.remove("STOPCAR")
-
-            # Setup INCAR to continue
-            incar = Incar.from_file("INCAR")
-            incar['ISTART'] = 1
-            incar.write_file("INCAR")
+        if self.auto_continue:
+            if os.path.exists("continue.json"):
+                actions = loadfn("continue.json").get("actions")
+                logger.info("Continuing previous VaspJob. Actions: {}".format(actions))
+                backup(VASP_BACKUP_FILES, prefix="prev_run")
+                VaspModder().apply_actions(actions)
 
-            shutil.copy('CONTCAR', 'POSCAR')
+            else:
+                # Default functionality is to copy CONTCAR to POSCAR and set
+                # ISTART to 1 in the INCAR, but other actions can be specified
+                if self.auto_continue is True:
+                    actions = [{"file": "CONTCAR", 
+                                "action": {"_file_copy": {"dest": "POSCAR"}}},
+                               {"dict": "INCAR",
+                                "action": {"_set": {"ISTART": 1}}}]
+                else:
+                    actions = self.auto_continue
+                dumpfn({"actions": actions}, "continue.json")
```

Comment:
I don't have a strong view about this. In fact, I am not even sure what the auto_continue is used for.